packages/cli/src/lib/upload/upload-command.unit.test.ts

```diff
@@ -1,23 +1,33 @@
-import { bundleRequire } from 'bundle-require';
 import { vol } from 'memfs';
-import { beforeEach, describe, expect, it } from 'vitest';
+import { describe, expect, it } from 'vitest';
 import { PortalUploadArgs, uploadToPortal } from '@code-pushup/portal-client';
+import { readCodePushupConfig } from '@code-pushup/core';
 import {
   ISO_STRING_REGEXP,
+  MEMFS_VOLUME,
   MINIMAL_REPORT_MOCK,
 } from '@code-pushup/testing-utils';
 import { DEFAULT_CLI_CONFIGURATION } from '../../../mocks/constants';
 import { yargsCli } from '../yargs-cli';
 import { yargsUploadCommandObject } from './upload-command';
 
+vi.mock('@code-pushup/core', async () => {
+  const { CORE_CONFIG_MOCK }: typeof import('@code-pushup/testing-utils') =
+    await vi.importActual('@code-pushup/testing-utils');
+  const core = await vi.importActual('@code-pushup/core');
+  return {
+    ...(core as object),
+    readCodePushupConfig: vi.fn().mockResolvedValue(CORE_CONFIG_MOCK),
+  };
+});
+
 describe('upload-command-object', () => {
   beforeEach(() => {
     vol.fromJSON(
       {
         'my-report.json': JSON.stringify(MINIMAL_REPORT_MOCK),
-        'code-pushup.config.ts': '', // only needs to exist for stat inside readCodePushupConfig
       },
-      '/test',
+      MEMFS_VOLUME,
```

Comment:
Candidate for a linting rule :)

Reply:
While there are ESLint rules for hardcoded values and repeated literals, they are disabled in the test preset I believe (cc @matejchalk) as hardcoding values in test is not necessarily wrong. Here I replaced it to be consistent with other files.

Reply:
We don't have a lint rule that covers this. Production code has [sonarjs/no-duplicate-string](https://github.com/SonarSource/eslint-plugin-sonarjs/blob/master/docs/rules/no-duplicate-string.md), but that's disabled in tests for good reason (DAMP vs DRY).